Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py

```diff
@@ -195,76 +416,175 @@ def fetch_indicators_command(client: Client) -> List[Dict[str, Any]]:  # pragma:
     Returns:
         list: indicator to populate in demisto server.
     """
-    owners = f'AND ({create_or_query("ownerName", demisto.getParam("owners"))}) '
-    tags = f'AND ({create_or_query("tags", demisto.getParam("tags"))}) '
-    status = f'AND ({create_or_query("status", demisto.getParam("status"))}) '
-    fields = set_fields_query(argToList(demisto.getParam("fields")))
+    indicators_url = set_url(params, 'indicators')
+    groups_url = set_url(params, 'groups')
+
+    indicators = []
+    groups = []
+    indicators_next_link = ''
+    groups_next_link = ''
+    while True:
+        if indicators_next_link or groups_next_link:
+            if indicators_next_link:
+                demisto.debug('Indicators Next Link: ' + indicators_next_link)
+                response, _, indicators_next_link = client.make_request(Method.GET,
+                                                                        url_suffix='',
+                                                                        get_next=True,
+                                                                        full_url=indicators_next_link)
+                indicators.extend(response)
+            if groups_next_link:
+                demisto.debug('Groups Next Link: ' + groups_next_link)
+                response, _, groups_next_link = client.make_request(Method.GET,
+                                                                    url_suffix='',
+                                                                    get_next=True,
+                                                                    full_url=groups_next_link)
+                groups.extend(response)
+        else:
```

Comment:
we will enter the else block when we finish with the API pagination (i.e. no next links) and make the same API call as the first one, no?